src/fix/main.c

```diff
@@ -144,8 +180,16 @@ static void printAcceptedMBCNames(void)
 	fputs("\tBANDAI_TAMA5 ($FD)\n", stderr);
 	fputs("\tHUC3 ($FE)\n", stderr);
 	fputs("\tHUC1+RAM+BATTERY ($FF)\n", stderr);
+
+	fputs("\n\tTPP1_1.x, TPP1_1.x+RUMBLE, TPP1_1.x+MULTIRUMBLE, TPP1_1.x+RTC,\n", stderr);
```

Comment:
Either make these `TPP1_1.0`, or have the first line be something like `TPP1_1.0 [supports versions 1.x]` or `TPP1_1.x [supports version 1.0]` (since only version 1.0 exists so far).

Reply:
Supporting 1.0 should be sufficient for the whole 1.x series according to SemVer, so I'm not sure what's wrong with this.

Reply:
This looks like it's saying to do `-m TPP1_1.x`; only the docs explain how the version number works. And since 1.0 is all there is so far, IMO it may as well specify that.

Reply:
I should point out that it's impossible for TPP1 to use SemVer because it doesn't have a revision/patch/whatever-you-call-the-third-thingy number.
(That being said, I do intend to use common sense when numbering versions if I ever get around to making a new one.)